lib/src/tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,129 @@
+import React from "react";
+import styled from "styled-components";
+import TooltipPropsType from "./types";
+import * as Tooltip from "@radix-ui/react-tooltip";
+import CoreTokens from "../common/coreTokens";
+
+const DxcTooltip = ({ position = "bottom", label, children }: TooltipPropsType): JSX.Element => {
+  const SIZE = 8;
+
+  return label ? (
+    <Tooltip.Provider delayDuration={300}>
+      <Tooltip.Root>
+        <Tooltip.Trigger asChild>
+          <TooltipTriggerContainer>{children}</TooltipTriggerContainer>
+        </Tooltip.Trigger>
+        <Tooltip.Portal>
+          <StyledTooltipContent side={position} sideOffset={SIZE} data-testid={"popover-content"}>
```

Comment:
I don't see the need to add a data-testid.

Reply:
Totally! I forgot to remove it.